conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -30,7 +30,7 @@ DELETE      /report-quarterly/income-and-expenses/view/test-only/all-schemas/pro
 
 + nocsrf
 GET         /report-quarterly/income-and-expenses/view/test-only/custom-login                         testOnly.controllers.CustomLoginController.showLogin
-GET         /hmrc-frontend/assets/hmrc-frontend-5.19.0.min.css                                        testOnly.controllers.CustomLoginController.showCss
+GET         /test-only/hmrc-frontend/assets/hmrc-frontend-5.19.0.min.css                              testOnly.controllers.CustomLoginController.showCss
```

Comment:
testOnly route will be removed from PROD (and I think from QA), so these CSS resources will be not available; might worth to revert this change

Reply:
If it might be needed in prod, is it worth moving this route out of the testOnly file?

Reply:
not sure what you mean, change above is to move out of prod route into testOnly, this would cause issues

Reply:
It's currently in the testOnly routes file. If it might be used in production, I think it might be more appropriate to move it into one of the other routes files
But taking a look at the method that the route calls, it calls the dynamic stub connector, so I'm not sure that it would be used in prod